extensions/wikia/CommunityPage/scripts/CommunityPageBenefitsPageviews.js

```diff
@@ -1,28 +1,17 @@
 require([
 	'wikia.pageviewsInSession',
 	'CommunityPageBenefitsModal',
-	'wikia.tracker',
-	'wikia.cookies'
-], function (pageviews, modal, tracker, cookies) {
+	'wikia.tracker'
+], function (pageviews, modal, tracker) {
 	'use strict';
 
 	function init() {
-		if (pageviews.getPageviewsCount() === 4) {
+		if (pageviews.getPageviewsCount() >= 4) {
 			modal.open();
-			setModalShownCookie();
 			trackModalImpression();
 		}
 	}
 
-	// This cookie is checked in CommunityPageSpecialHooks::onBeforePageDisplay to avoid unnecessary script loading
-	function setModalShownCookie() {
-		cookies.set('cpBenefitsModalShown', 1, {
-			domain: mw.config.get('wgCookieDomain'),
-			expires: 2592000000, // 30 days
-			path: mw.config.get('wgCookiePath')
-		});
-	}
-
 	function trackModalImpression() {
```

Comment:
Does this method need to be invoked here? I think that it belongs to CommunityPageBenefitsModal